devise_2fa.gemspec

```diff
@@ -1,11 +1,12 @@
-# -*- encoding: utf-8 -*-
-lib = File.expand_path('../lib', __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'devise-2fa/version'
+$:.push File.expand_path("lib", __dir__)
 
+# Maintain your gem's version:
+require "devise_2fa/version"
+
+# Describe your gem and declare its dependencies:
 Gem::Specification.new do |gem|
   gem.name          = 'devise-2fa'
-  gem.version       = Devise::TwoFactor::VERSION
+  gem.version       = Devise2fa::VERSION
```

Comment:
What was the motivation to change this constant name  (see line 4 too)?
I think this PR should change as MINIMAL as possible. Not renaming this OSS's constant names